pipelines/skylab/optimus/Optimus.wdl

```diff
@@ -56,7 +59,7 @@ workflow Optimus {
 
   # version of this pipeline
 
-  String pipeline_version = "4.2.1"
+  String pipeline_version = "4.3.0"
```

Comment:
I wonder if this is a patch update since the input is optional and defaulted to have the same function as it did before?